### ISSUE TITLE
Reset wifi configuration, switch back to setup mode

### ISSUE THREAD

**Issue by dotleon** [tomikaa87/gree-remote#49]
Is there any way I can issue a command that resets the wifi configuration?

When setting up, I didn't realize some devices in the wifi list are not mine, so I have some devices connecting to my WiFi AP. I can control some unknown AC's that are not in my house. They accept commands, and I can confirm that they to turn on/off, etc.

I've tried to send them

` {"psw": "00000000","ssid": "00000000","t": "wlan"}`

but of course they won't accept it since they're not in setup mode anymore.

I rather not change my SSID and password, as there are too many IoT devices connected to it, some have the pass cooked into the firmware, etc, so I really hope I can just reset them remotely. I could also probably blacklist their MAC address, but again, they'd keep trying to connect indefinitely I think.

There's also the go around my neighbourhood asking about their ACs, or look for GREE external units, then explain them how to reset, etc option but hopefully it doesn't have to come to that.

**Comment by Daniel-K1986** [comment]
Hi,

yesterday evening I have tried to figure out the same. Unfortunately it did not work out. I have also tried to send the package you described to the IP of the AC.
In another manual (for Sinclair - not for Gree) I have found, that you can reset the wifi settings completely by pressing "Wifi + Mode" on the remote control. This did the trick for me (unless you, I have access to the AC - but had to change the AP). After a few seconds, the wifi AP of the AC was up again.
So I assume, that there is no remote way to reset this. Sorry.

Kind regards,

Daniel

**Comment by dotleon** [comment]
I was pondering on building battery powered IR grenades I could throw through some open windows, or a very powerful IR blaster hoping the signal would bounce around enough, but I took a walk around the neighbourhood instead. I found a neighbour with multiple gree external units. I've explained the situation and they managed to reset their ACs.

As I think this could only be used in some very specific edge cases like mine, I close the issue.
